klasses/models.py

```diff
@@ -60,6 +60,9 @@ class BootcampRun(models.Model):
     run_key = models.IntegerField(unique=True, db_index=True)
     start_date = models.DateTimeField(null=True)
     end_date = models.DateTimeField(null=True)
+    bootcamp_run_id = models.CharField(
+        null=True, unique=True, blank=True, max_length=255
```

Comment:
By keeping `blank=True` with `unique=True`, do you know it will keep blank as unique and it would not let create more than one blank entry?